staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -766,6 +766,11 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdownBasedOnPodPriority
 	// +optional
 	ShutdownGracePeriodByPodPriority []ShutdownGracePeriodByPodPriority `json:"shutdownGracePeriodByPodPriority,omitempty"`
+	// crashLoopBackOff contains config to modify node-level parameters for
+	// container restart behavior
+	// +featureGate=EnableCrashLoopBackOffMax
+	// +optional
+	CrashLoopBackOff *CrashLoopBackOffConfig `json:"crashloopbackoff,omitempty"`
```

Comment:
why is the JSON name "crashloopbackoff" and not "crashLoopBackOff" ?  Is that on purpose?

Reply:
It was, I was feeling it was kinda like DNS->dns or IPTables->iptables since it's "named after" an error, but I'm not attached to that and becoming less confident that kind of style really applies here since it's not a Go variable name.

Also to throw more naming wrenches in I feel like everything I've sprinkled around named after `CrashLoopBackOff` could instead take as its namesake some riff on "container restart" as the more general feature space, like "containerRestartConfig" or something.

Reply:
For now fixed as `CrashLoopBackOff` in 6692dda